### DISCUSSION TITLE
Add Captures button on top menù (after/before Pro)

### DISCUSSION THREAD

**Discussion by DaRkViVi** [ChristopherKlay/StadiaEnhanced#69] [Ideas & Requests]
I usually find "hard" to reach the captures page to download/share screenshots and videos, and the feature to add them at the bottom of the Home Page is nice but still "slow" to reach.

I think it would be nice to add a "Captures" button on top menù to reach the page easily.

**Comment by ChristopherKlay** [comment]
We actually had it in the navbar in a very early version, but you can also find your captures by just hitting the little down arrow at the top right (settings menu).

The very first entry in the list, links to your captures (:

**Reply by DaRkViVi** [reply]
Oh, guess I've missed it. Thanks!
